events/0515_tdd.html

```diff
@@ -0,0 +1,164 @@
+
+    <link rel="stylesheet" href="css/reveal.css">
+    <link rel="stylesheet" href="css/theme/gdiaa.css" id="theme">
+    <link rel="stylesheet" href="lib/css/rainbow.css">
+    <link rel="stylesheet" href="css/print/pdf.css" type="text/css" media="print">
+
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
+    <script src="lib/js/head.min.js"></script>
+
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+            <!-- Event Intro -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to TDD</h3>
+                    <small>
+                        <div class="box--small">Vera Reynolds &nbsp; | &nbsp; @vecho_ralfa</div>
+                        <div class="box--small copy--small">@gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #IntroTDD</div>
+                    </small>
+                </div>
+                <aside class="notes"></aside>
+            </section>
+
+            <!-- GDI Welcome -->
+            <section>
+                <h3>Welcome!</h3>
+                <!-- <p class="box"><strong>Girl Develop It</strong> is here to empower and assist women interested in getting into and advancing in the tech industry through affordable and accessible programs.</p> -->
+                <p class="box"><strong>Girl Develop It</strong> is here to provide affordable programs for adult women interested in learning web and software development in a judgment-free environment.</p>
+
+                <div class="copy--small">
+                    <h4 class="green">Some "rules"</h4>
+                    <ul>
+                        <li>We are here for you!</li>
+                        <li>Every question is important.</li>
+                        <li>Help each other.</li>
+                        <li><strong>Have fun!</strong></li>
+                    </ul>
+                </div>
+
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harrasment free learning experience for <em>everyone</em>.</strong><br/>For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title"><span class="blue">1-year Anniversary Fundraiser</span></h3>
+                    <h3 class="event__title"><small>Thursday 6/4 @ The Forge (Pillar)</small></h3>
+                    <p><small>6pm - 9pm &nbsp; | &nbsp; 301 E Liberty St, Ann Arbor</small></p>
```

Comment:
can you add the http://bit.ly/gdia2-anniversary link to the bottom of this slide